src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -11001,6 +11113,64 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/ActionBodyExecTemplate'
+  RuleSetPaging:
+    type: object
+    required:
+      - list
+    properties:
+      list:
+        type: object
+        required:
+          - pagination
+          - entries
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/RuleSetEntry'
+  RuleSetEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/RuleSet'
+  RuleSet:
+    type: object
+    required:
+      - id
+      - ownedBy
+    properties:
+      id:
+        description: Identifier for the rule set
+        type: string
+      ownedBy:
+        description: The identifier of the folder which owns the rule set
+        type: string
+      sharedWith:
+        description: List of folder identifiers which also use the rule set
```

Comment:
Minor: We have terms such as "sharedWith" and "use" here. Why not use the word link in both places, eg. "linkedBy" and "link".

Reply:
> requires ?include=usedBy
Why do we need to hide it behind the include query parameter? I can understand this under nodes, since nodes have many different properties, but here we only have 3.

Reply:
Ok I will put it in the includes then